develop/tutorials/articles/03-developing-a-web-application/06-displaying-messages-errors/02-adding-error-messages.markdown

```diff
@@ -1,21 +1,20 @@
 # Adding Failure and Success Messages  
 
-To display the correct feedback to users properly, you must edit your portlet
-classes to make use of @product@'s `SessionMessages` and `SessionErrors` classes.
-These classes collect feedback messages that are shown to the user in the view
-layer using a simple tag. 
+To properly display correct feedback to users, you must edit your portlet 
```

Comment:
@ngaskill I likely edited this from the original to read the way I had it because of the split infinitive. Whenever I see one, I think of the title of an old Mad magazine Star Trek parody: To boldly go where no one has split an infinitive before. :-)